From 70933c73a14f079dbd0febc39b9448a0d772cd29 Mon Sep 17 00:00:00 2001
From: Vincent Knecht <vincent.knecht@mailoo.org>
Date: Tue, 23 Nov 2021 09:14:30 +0100
Subject: [PATCH 205/210] FROMLIST: dt-bindings: input: touchscreen: msg2638:
 Document msg2138 support

Document msg2138 support by adding mstar,msg2138 compatible.

Signed-off-by: Vincent Knecht <vincent.knecht@mailoo.org>
Acked-by: Rob Herring <robh@kernel.org>
---
 .../devicetree/bindings/input/touchscreen/mstar,msg2638.yaml  | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/input/touchscreen/mstar,msg2638.yaml b/Documentation/devicetree/bindings/input/touchscreen/mstar,msg2638.yaml
index 3a42c23faf6..2fb7e01bb65 100644
--- a/Documentation/devicetree/bindings/input/touchscreen/mstar,msg2638.yaml
+++ b/Documentation/devicetree/bindings/input/touchscreen/mstar,msg2638.yaml
@@ -14,7 +14,9 @@ allOf:
 
 properties:
   compatible:
-    const: mstar,msg2638
+    enum:
+      - mstar,msg2138
+      - mstar,msg2638
 
   reg:
     const: 0x26
-- 
2.34.1

